frankenphp/docker-entrypoint.sh

```diff
@@ -22,7 +22,7 @@ if [ "$1" = 'frankenphp' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 		fi
 	fi
 
-	if [ ! -d 'vendor/' ]; then
+	if [ -z "$(ls -A 'vendor/')" ]; then
```

Comment:
Wouldn't the script fail the directory doesn't exist?

Reply:
No, if the `vendor` directory does not exist, the condition will be satisfied

> This condition will be true if the `vendor` directory does not exist (in which case the `ls` command will return an error and the command output will be empty) or if there are no files or subdirectories in the directory (in which case `ls -A` will output empty lines, and the output of the command will also be empty)

Issue #490

Reply:
Tested (locally), the syntax should be:
```suggestion
	if [ -z "$(ls -A 'vendor/' 2>/dev/null)" ]; then
```